autoscaling-optional.tf

```diff
@@ -36,7 +34,7 @@ resource "aws_cloudwatch_metric_alarm" "cpu_high" {
   threshold           = var.cpu_utilization_high_threshold_percent
 
   dimensions = {
-    AutoScalingGroupName = join("", aws_autoscaling_group.default.*.name)
+    AutoScalingGroupName = join("", aws_autoscaling_group.this.*.name)
```

Comment:
We don't need to impose our coding conventions on a public fork--it's probably more important to minimize the differences in this case.